src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
```

Comment:
Ensure that the path 'fonts/Avenir-Book.ttf' is correct and that the font file is accessible. This is crucial for the custom font to load properly.